lib/npm/install.js

```diff
@@ -14,6 +14,9 @@ function install(context, next) {
   var args = ['install', '--loglevel', context.options.npmLevel];
   var bailed = false;
   context.emit('data', 'info', 'npm:', 'install started');
+  for (var i in context.module.envVar) {
```

Comment:
this should be moved into `lib/create-options.js`

that way we do not need to repeat the iteration code and can simply test the output of the test options function